arch/arm64/boot/dts/qcom/sdm630-sony-xperia-nile.dtsi

```diff
@@ -674,6 +674,20 @@
 		bias-disable;
 		drive-strength = <2>;
 	};
+
+	panel_reset_gpio: panel-reset-gpio {
+		pins = "gpio53";
+		function = "gpio";
+		drive-strength = <8>;
+		bias-disable = <0>;
+	};
+
+	panel_te_gpio: panel-te-gpio {
```

Comment:
nitpicking, but dtbs_check warns about this:

If you scroll up and see how DT nodes are named (not labels!) you can see that they must end with `-state`. So,
```suggestion
	panel_te_gpio: panel-te-gpio-state {
```
and also `panel_reset_gpio: panel-reset-gpio-state {` for the node above this. Since sony-nile DTs are already upstream, we probably should try to not increase warnings count if possible... Also I can fix this later when merging